tests/www/test_utils.py

```diff
@@ -103,14 +105,45 @@ def test_params_search(self):
         self.assertEqual('search=bash_',
                          utils.get_params(search='bash_'))
 
-    def test_params_showPaused_true(self):
-        """Should detect True as default for showPaused"""
-        self.assertEqual('',
-                         utils.get_params(showPaused=True))
-
-    def test_params_showPaused_false(self):
-        self.assertEqual('showPaused=False',
-                         utils.get_params(showPaused=False))
+    @parameterized.expand([
+        (True, False, ''),
+        (False, True, ''),
+        (True, True, 'showPaused=True'),
+        (False, False, 'showPaused=False'),
+        (None, True, ''),
+        (None, False, ''),
+    ])
+    def test_params_showPaused(self, show_paused, hide_by_default, expected_result):
+        os.environ['AIRFLOW__WEBSERVER__HIDE_PAUSED_DAGS_BY_DEFAULT'] = str(hide_by_default)
+
+        try:
```

Comment:
Why did you enclose it in a `try-finally` block?

Reply:
If it is only because of the `ModuleNotFoundError: No module named 'tests.test_utils'` then you can remove it, because mik-laj fixed it.

Reply:
@feluelle This is to set a custom value for `AIRFLOW__WEBSERVER__HIDE_PAUSED_DAGS_BY_DEFAULT` environment variable before each test and tear it down after the test is done.

Reply:
Okay, but I mean why should `self.assertEqual(expected_result, utils.get_params(showPaused=show_paused))` raise an error?

Wouldn't..
```
self.assertEqual(expected_result, utils.get_params(showPaused=show_paused))
del os.environ['AIRFLOW__WEBSERVER__HIDE_PAUSED_DAGS_BY_DEFAULT']
```
..also work?

Reply:
Rather than setting the env var can you use the `conf_vars` context manager we use in other tests (check out `./tests/test_utils/config.py`)

Reply:
> Okay, but I mean why should self.assertEqual(expected_result, utils.get_params(showPaused=show_paused)) raise an error?
Wouldn't..
self.assertEqual(expected_result, utils.get_params(showPaused=show_paused))
del os.environ['AIRFLOW__WEBSERVER__HIDE_PAUSED_DAGS_BY_DEFAULT']
..also work?

@feluelle if for some reason `utils.get_params` method raises an error the `del os.environ['AIRFLOW__WEBSERVER__HIDE_PAUSED_DAGS_BY_DEFAULT'] would not get executed leaving that environment enabled for other tests. That was a teardown that only applies to a single test.

> Rather than setting the env var can you use the conf_vars context manager we use in other tests (check out ./tests/test_utils/config.py)

@ashb I'm taking a look. Thanks for the pointer!

Reply:
@alrolorojas Thank you - makes sense. But yes the context manager would be probably nicer to use 👍